src/Cedar/VLanUnix.c

```diff
@@ -815,6 +846,9 @@ void UnixVLanFree()
 		UNIX_VLAN_LIST *t = LIST_DATA(unix_vlan, i);
 
 		UnixCloseTapDevice(t->fd);
+#ifdef	UNIX_BSD

```

Comment:
Tab instead of space.

Reply:
It's tab here already. But I mixed space and tab. Which should I use?

Reply:
Spaces for defines.

Reply:
Oops, I have just pushed tabs. Will repush with spaces.